packages/server/lib/plugins/child/run_require_async_child.js

```diff
@@ -110,25 +96,16 @@ function run (ipc, file, projectRoot) {
     debug('User is loading an ESM config file')
 
     try {
-      debug('Trying to use esbuild to run their config file.')
```

Comment:
I've removed this for now, we can definitely integration w/ the user's esbuild, but I think that should be done separately, since there's some unknowns.

I've called out this code deletion and referenced it in the `esbuild` issue so we don't forget about it: https://github.com/cypress-io/cypress/issues/22074